src/SplitPackages/Frameworks.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using NuGet.Frameworks;
+
+namespace SplitPackages
+{
+    public static class Frameworks
+    {
+        public static string NetStandardApp15 => FrameworkConstants.CommonFrameworks.NetStandardApp15.DotNetFrameworkName;
```

Comment:
This you need to also add `NetCoreApp10` now. Our executables will be switching to this tfm very soon.